server/routes/dashBoard.js

```diff
@@ -0,0 +1,251 @@
+const express = require("express");
+const router = express.Router();
+const { updateData, decodeToken } = require("../util/util");
+
+// Models;
+const { Task, Column, Dashboard } = require("../models/Dashboard");
+
+//@CreateBoard
+router.post("/getDashboard", async (req, res) => {
```

Comment:
Shouldn't this be a .get request?
Also try to use rest naming conventions https://restfulapi.net/resource-naming/

Reply:
Thank you for reviewing!
- changed post to get.
- changed route name /dashboards/dashboard. it is used to /dashboard/getDashboard